llpc/CMakeLists.txt

```diff
@@ -384,6 +370,7 @@ add_dependencies(amdllpc llpc_standalone_compiler)
 target_link_libraries(amdllpc PRIVATE llpc_standalone_compiler)
 set_compiler_options(amdllpc ${LLPC_ENABLE_WERROR})
 add_compile_definitions(amdllpc PRIVATE SH_EXPORTING)
+target_link_libraries(amdllpc PRIVATE spvgen_static)
```

Comment:
This line seems to be added by mistake. It was removed in https://github.com/GPUOpen-Drivers/llpc/commit/b750345#diff-7b9725e04fc088aab06b9273d5399b456c4bdb52d536c1869ae16be6801e4db3
Looks like that part of the diff did not make it to the internal branch.

Reply:
> This line seems to be added by mistake. It was removed in [b750345#diff-7b9725e04fc088aab06b9273d5399b456c4bdb52d536c1869ae16be6801e4db3](https://github.com/GPUOpen-Drivers/llpc/commit/b750345#diff-7b9725e04fc088aab06b9273d5399b456c4bdb52d536c1869ae16be6801e4db3) Looks like that part of the diff did not make it to the internal branch.

Thanks, will remove it.